File: src/targets/python/requests/fixtures/full.py
import requests

url = "https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value"

payload = { "foo": "bar" }
headers = {
    "cookie": "foo=bar; bar=baz",
    "accept": "application/json",
    "content-type": "application/x-www-form-urlencoded"
}

response = requests.post(url, data=payload, headers=headers)

print(response.json())

File: src/targets/swift/urlsession/fixtures/full.swift
import Foundation
#if canImport(FoundationNetworking)
  import FoundationNetworking
#endif

let headers = [
  "cookie": "foo=bar; bar=baz",
  "accept": "application/json",
  "content-type": "application/x-www-form-urlencoded"
]

let postData = Data("foo=bar".utf8)

var request = URLRequest(url: URL(string: "https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value")!)
request.httpMethod = "POST"
request.allHTTPHeaderFields = headers
request.httpBody = postData

let (data, response) = try await URLSession.shared.data(with: request)
print(String(decoding: data, as: UTF8.self))
